### PR TITLE
✨Add Alias for Motor Gear Cartridge

### LINKED ISSUE #387: RFC: Add aliases for Motor Gear cartridges enum w/ Colors or RPM (PROS 3)
Currently, our Motor Gear cartridge definition structure is based on the radio (a 200 motor cartridge uses the enum E_MOTOR_GEARSET_18). Based on community lingo these days, I think we should switch it to be some form of:

E_MOTOR_GEARSET_RED
E_MOTOR_GEAR_RED
E_MOTOR_GEAR_100

Comments? I think all of these options are good and we could consider using all 3 of these for the community's ease of use.

### PR DESCRIPTION
**Pull request by aaryan-gautam**
#### Summary:

Added aliases for motor gearset with color and rpm

#### Motivation:

The aliases are added in the enum motor_gearset_e and given the same value as the corresponding gearset
##### References (optional):

#Closes #387
#### Test Plan:

Run checks
- [x] test item

### CODE REVIEW

File: include/pros/motors.h [hunk 1]
```diff
@@ -597,11 +597,18 @@ typedef enum motor_encoder_units_e {
...
+	E_MOTOR_GEAR_RED = 0,
+	E_MOTOR_GEAR_100 = 0,
 	E_MOTOR_GEARSET_18 = 1,  // 18:1, 200 RPM, Green gear set
+	E_MOTOR_GEAR_GREEN = 1,
+	E_MOTOR_GEAR_200 = 1,
 	E_MOTOR_GEARSET_06 = 2,  // 6:1, 600 RPM, Blue gear set
+	E_MOTOR_GEAR_BLUE  = 2,
+	E_MOTOR_GEAR_600 = 2,
 	E_MOTOR_GEARSET_INVALID = INT32_MAX
 } motor_gearset_e_t;
```
Comment (WillXuCodes) [inline include/pros/motors.h:610]:
```suggestion
```
Remove newline

File: include/pros/motors.h [hunk 2]
```diff
@@ -597,11 +597,18 @@ typedef enum motor_encoder_units_e {
...
 
+
 #ifdef PROS_USE_SIMPLE_NAMES
```
Comment (WillXuCodes) [inline include/pros/motors.h:612]: Be sure to update simple names with your added enum members.

File: include/pros/motors.h [hunk 3]
```diff
@@ -597,11 +597,18 @@ typedef enum motor_encoder_units_e {
...
 typedef enum motor_gearset_e {
 	E_MOTOR_GEARSET_36 = 0,  // 36:1, 100 RPM, Red gear set
+	E_MOTOR_GEAR_RED = 0,
+	E_MOTOR_GEAR_100 = 0,
 	E_MOTOR_GEARSET_18 = 1,  // 18:1, 200 RPM, Green gear set
+	E_MOTOR_GEAR_GREEN = 1,
+	E_MOTOR_GEAR_200 = 1,
 	E_MOTOR_GEARSET_06 = 2,  // 6:1, 600 RPM, Blue gear set
+	E_MOTOR_GEAR_BLUE  = 2,
+	E_MOTOR_GEAR_600 = 2,
```
Comment (WillXuCodes) [inline include/pros/motors.h:607]:
```suggestion
	E_MOTOR_GEARSET_36 = 0,  // 36:1, 100 RPM, Red gear set
	E_MOTOR_GEAR_RED = E_MOTOR_GEARSET_36,
	E_MOTOR_GEAR_100 = E_MOTOR_GEARSET_36,
	E_MOTOR_GEARSET_18 = 1,  // 18:1, 200 RPM, Green gear set
	E_MOTOR_GEAR_GREEN = E_MOTOR_GEARSET_18,
	E_MOTOR_GEAR_200 = E_MOTOR_GEARSET_18,
	E_MOTOR_GEARSET_06 = 2,  // 6:1, 600 RPM, Blue gear set
	E_MOTOR_GEAR_BLUE  = E_MOTOR_GEARSET_06,
	E_MOTOR_GEAR_600 = E_MOTOR_GEARSET_06,
```

makes maintenance slightly easier

**Review by WillXuCodes — CHANGES_REQUESTED** [review]
Requested changes made

**Comment by WillXuCodes** [comment]
can't do it on mobile, but could you fix the title of the PR? Thanks.

**Comment by aaryan-gautam** [comment]
> can't do it on mobile, but could you fix the title of the PR? Thanks.

Done

**Review by WillXuCodes — APPROVED** [review]
saucin

**Pull request merged**